src/main/resources/templates/Answer.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html SYSTEM "http://www.thymeleaf.org/dtd/xhtml1-strict-thymeleaf-spring4-4.dtd">
+<html lang="en"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Getting Started: Serving Web Content</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body>
+
+<h1 th:text="${surveyName}"></h1>
+
+<div th:with="expr_result = ${empty_or_not}">
+<!--  if survey had no questions -->
+  <div th:if="${expr_result}">
+    Survey has no questions
+    <br>
+    <br>
+    <button onclick="window.location.href='http://localhost:8080/';">
```

Comment:
replace with link
also don't do the local host thing. You can do what you did below with the @{} thing